.github/workflows/test_linux_packages.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2025 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Test Linux Packages
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - ADHOCBUILD
+  workflow_call:
+
+jobs:
+  test_rocm_info:
+    name: "Sanity ROCM Test"
+    runs-on: linux-mi300-1gpu-ossci
+    strategy:
+      fail-fast: true
+    defaults:
+      run:
+        shell: bash
```

Comment:
```suggestion
```
`bash` is the default on non-Windows systems and we normally only set this in workflows that are intended to run on different operation systems. Should it could be dropped here.